common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3899,6 +3901,14 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(true)
 
+  val CLIENT_BATCH_HANDLE_CHANGE_PARTITION_PARALLELISM: ConfigEntry[Int] =
+    buildConf("celeborn.client.shuffle.batchHandleChangePartition.parallelism")
```

Comment:
Maybe `celeborn.client.shuffle.batchHandleChangePartition.partitionBuckets` is better?

Reply:
updated